xml/containers-build.xml

```diff
@@ -26,32 +26,29 @@
    <title>Docker</title>
    <para>
     Docker is a system for creating and managing containers. Its core is the
-    Docker Open Source Engine&mdash;a lightweight virtualization solution to
-    run containers simultaneously on a single host. Docker containers are
-    defined using Dockerfiles. A Dockerfile describes how to assemble a private
-    filesystem for a container. The file can also contain metadata that
-    describes how to run the container. Most Dockerfiles start from a parent
-    image. For an introduction on how to create custom base &sle; Docker
-    images, see <xref linkend="cha-docker-overview"/>.
+    Docker Open Source Engine&mdash;a lightweight virtualization solution to run
+    containers simultaneously on a single host. Docker containers are built
+    using <xref linkend="Dockerfile"/> Dockerfiles. For a genetal introduction
+    to &deng;, refer to <xref linkend="cha-docker-overview"/>.
    </para>
   </sect2>
 
   <sect2 xml:id="sec-build-tools-podman">
    <title>Podman</title>
    <para>
-    Podman is short for Pod Manager tool, and its purpose extends beyond just
+    Podman is short for Pod Manager tool, and its purpose extends beyond
```

Comment:
I'd remove the `and its purpose extends beyond building images.`, as that's also not part of the `Docker` section above.